src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -34,11 +35,13 @@ class ErrorBoundary extends React.Component<Props, State> {
     }
 
     render() {
-        if (this.state.error) {
-            return this.props.component;
+        const { errorComponent: ErrorComponent, children } = this.props;
+        const { error } = this.state;
+        if (error) {
+            return <ErrorComponent error={error} />;
         }
 
-        return this.props.children;
+        return children;
```

Comment:
If the value is only used once locally, is there a reason to destructure it at the top of the method?